<ui version="4.0" >
 <class>MatchRasterDialog</class>
 <widget class="QDialog" name="MatchRasterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find a font using raster sample</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="2" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" colspan="4" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Input Image</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QCheckBox" name="grabModeBox" >
              <property name="text" >
               <string>Grab mode</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSlider" name="grabZoom" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip" >
               <string>Zoom level</string>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
              <property name="maximum" >
               <number>10</number>
              </property>
              <property name="sliderPosition" >
               <number>1</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition" >
               <enum>QSlider::TicksAbove</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLineEdit" name="imagePath" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseButton" >
              <property name="text" >
               <string>&amp;Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="4" >
        <widget class="QGroupBox" name="sampleBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>3</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Raster sample preview</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" >
          <item rowspan="2" row="1" column="0" >
           <widget class="IView" name="iView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>4</horstretch>
              <verstretch>4</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2" >
           <widget class="QWidget" native="1" name="widget" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_5" >
             <item row="1" column="0" >
              <layout class="QFormLayout" name="formLayout" >
               <item row="0" column="0" >
                <widget class="QLabel" name="label_2" >
                 <property name="text" >
                  <string>C&amp;haracter:</string>
                 </property>
                 <property name="buddy" >
                  <cstring>letter</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" >
                <widget class="QLineEdit" name="letter" >
                 <property name="sizePolicy" >
                  <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip" >
                  <string>A unique character or a four digits hexadecimal Unicode value</string>
                 </property>
                 <property name="maxLength" >
                  <number>4</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2" >
              <widget class="QPushButton" name="searchButton" >
               <property name="text" >
                <string>&amp;Search</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <spacer name="horizontalSpacer_4" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2" >
              <widget class="QCheckBox" name="checkInteractive" >
               <property name="toolTip" >
                <string>If set, you will be prompted for each matching font</string>
               </property>
               <property name="text" >
                <string>Interactive</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="label_3" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Click on the letter to let Fontmatrix compute the bounding box</string>
            </property>
            <property name="scaledContents" >
             <bool>false</bool>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" >
           <widget class="QCheckBox" name="tweakRectBox" >
            <property name="text" >
             <string>Tweak rect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" name="gridLayout_6" >
       <item row="0" column="2" >
        <widget class="QLabel" name="fontName" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4" >
        <widget class="QProgressBar" name="progressBar" >
         <property name="value" >
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Compare with:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4" >
        <widget class="IView" name="compView" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3" >
        <widget class="QWidget" native="1" name="questionWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_7" >
          <item row="0" column="0" colspan="2" >
           <widget class="QLabel" name="scoreLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QDialogButtonBox" name="buttonBox" >
            <property name="standardButtons" >
             <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
            </property>
            <property name="centerButtons" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QPushButton" name="stopButton" >
            <property name="text" >
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>424</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IView</class>
   <extends>QGraphicsView</extends>
   <header>iview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MatchRasterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>335</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel" >
     <x>339</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grabModeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>browseButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>234</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>277</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grabModeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>imagePath</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>216</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>234</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grabModeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>grabZoom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>201</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
